js/router.js

```diff
@@ -78,7 +78,7 @@ export default class ObRouter extends Router {
   }
 
   user(guid, tab, ...args) {
-    tab = tab || 'store'; // eslint-disable-line no-param-reassign
+    tab = tab || 'Store'; // eslint-disable-line no-param-reassign
```

Comment:
The tab value is taken from the route and we don't want it capitalized there (e.g. we don't want @themes/Store), so I don't think we should be capitalizing the tab here.

I know the user view needs it capitalized because it needs to map to a certain data structure in the view, but that is very much an implementation detail of the user view. Can we just have the userView capitalize it when it gets it?